code/datums/components/shovel_hands.dm

```diff
@@ -0,0 +1,42 @@
+/// This component lets mobs dig up the floor with their bare hands
+/datum/component/shovel_hands
+	dupe_mode = COMPONENT_DUPE_SOURCES
+	/// It's a lie, they're actually just using a shovel
+	var/obj/item/shovel/internal_shovel
```

Comment:
lol

Reply:
if we had something like `TOOL_SCREWDRIVER` or `TOOL_WRENCH` for behavior we wouldn't even need this but that is shrimply too much work to implement so w/e